### PR TITLE
docs : store listing designs

### LINKED ISSUE #110: Design Play Store Cover Poster
This issue is to create an eye-catching and professional Play Store cover poster for our application. The poster should effectively convey the app's purpose and features while maintaining consistent branding elements.

### LINKED ISSUE #111: Design Play Store Sample Screenshots
Design high-quality sample screenshots for Play Store listing. The screenshots should showcase the app's user interface, key functionalities, and various use cases to attract potential users.

### PR DESCRIPTION
**Pull request by devmusiczz**
- I added the cover poster of RESONATE with corrections.
- added the Play Store sample screenshots including paring screenshot.

Closes #110 
Closes #111

### CODE REVIEW

**Comment by devmusiczz** [comment]
closes #109 #110 #111

File: docs/store_listing/images/PLAYSTORE SCREENSHOTS/4.png
Comment (chandansgowda) [inline docs/store_listing/images/PLAYSTORE SCREENSHOTS/4.png:1]: My email is visible in this. Can you please use a different screenshot.

File: docs/store_listing/images/PLAYSTORE SCREENSHOTS/2.png
Comment (chandansgowda) [inline docs/store_listing/images/PLAYSTORE SCREENSHOTS/2.png:1]: The app bar seems to be congested in this image. Correct it

**Review by chandansgowda — CHANGES_REQUESTED** [review]

File: docs/store_listing/images/PLAYSTORE SCREENSHOTS/2.png
Comment (chandansgowda) [inline docs/store_listing/images/PLAYSTORE SCREENSHOTS/2.png:1]: I think the image is stretched now.

**Review by chandansgowda — CHANGES_REQUESTED** [review]

**Comment by chandansgowda** [comment]
Any updates @devmusicz ?

File: docs/store_listing/description.md
Comment (chandansgowda) [inline docs/store_listing/description.md:1]: The content looks generated and most of it is not related to Resonate. Please go through the PRs and modify or remove the file and take it up as a new issue.

**Review by chandansgowda — CHANGES_REQUESTED** [review]
Please rename PLAYSTORE SCREENSHOTS to playstore_screenshots

**Comment by chandansgowda** [comment]
Great Job @devmusicz 
Sorry. This PR cannot be merged under hacktoberfest since it was raised 3 weeks ago. Please feel free to work on other issues or a feature you would like to implement.

**Pull request merged**